ceres.py

```diff
@@ -727,6 +731,7 @@ def write(self, sequence):
         #   self.fsPath, byteOffset, filesize, sequence)
         raise
       fileHandle.write(packedValues)
+      fileHandle.close()
```

Comment:
I'm not so convinced that RAII works like that in Python.  I've ran some other programs that use ceres in a tight loop and still hits `Errno 24`, which would have me believe that `with open` is an eventual guarantee, not an explicit one.

I can do some tests for that though.  But wouldn't it depend on `__exit__` being implemented properly by the object returned from `open()`?